src/routes/commands/commands_resource.ts

```diff
@@ -118,26 +117,26 @@ export const commands: Command[] = [
     },
     {
         name: "Softban",
-        tooltip: "Softbans the specified user",
+        tooltip: "Bans, then unbans, the specified user, deleting their recent messages",
         quick: "-mod softban @Samoxive#1234 Spamming nsfw pictures",
         description:
-            "Softbans the mentioned user, banning and unbanning them afterwards, effectively kicking them. This deletes " +
-            "the specified user's messages in the last x days (x is 1 by default, can be 7 at maximum). " +
-            "If a text is given after the mention, that text is used as the softban reason. To customize " +
-            'the day count of messages deleted, you can use the "|" seperator and specify the number afterwards.',
+            "Bans the specified user, then unbans them afterwards, effectively kicking them. This deletes " +
+            "the specified user's messages in the last 1-7 days, default 1. " +
```

Comment:
Could be phrased "defaults to the last one day"?

Reply:
Will fix